tel2puml/find_unique_graphs/otel_ingestion/otel_data_source.py

```diff
@@ -27,3 +62,121 @@ def __next__(self) -> OTelEvent:
         :rtype: :class: `OTelEvent`
         """
         pass
+
+
+class JSONDataSource(OTELDataSource):
+    """Class to handle parsing JSON OTel data from a file or directory,
+    returning OTelEvent objects.
+    """
+
+    def __init__(self) -> None:
```

Comment:
I think the init for this can be a TypedDict class for the config fields under json in the yaml